### PR TITLE
Renaming RWTH Aachen to RWTH Aachen University. Additionally adding non-listed faculty members.

### PR DESCRIPTION
**Pull request by LKlinke**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-c.csv [hunk 1]
```diff
@@ -1008,6 +1008,7 @@ Christopher Mears,Monash University,http://monash.edu/research/explore/en/person
 Christopher Michael Homan,Rochester Institute of Technology,https://www.cs.rit.edu/~cmh,kU6puLcAAAAJ
 Christopher Musco,New York University,https://www.chrismusco.com,HXXSrNMAAAAJ
 Christopher Nitta,Univ. of California - Davis,https://web.cs.ucdavis.edu/~cjnitta,vEPWWu0AAAAJ
+Christopher Morris,RWTH Aachen University,https://chrsmrrs.github.io/,3f6_I8MAAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv]: Missing disambiguation suffix: https://dblp.org/pid/156/7303.html

File: csrankings-p.csv [hunk 2]
```diff
@@ -122,7 +122,7 @@ Paola Quaglia,University of Trento,https://webapps.unitn.it/du/en/Persona/PER000
 Paola Velardi,Sapienza University of Rome,https://sites.google.com/a/di.uniroma1.it/paolavelardi,yf0g6zkAAAAJ
 Paolo A. G. Sivilotti,Ohio State University,http://web.cse.ohio-state.edu/~paolo,NOSCHOLARPAGE
 Paolo Baldan,University of Padova,https://www.math.unipd.it/~baldan,mPd-c8YAAAAJ
-Paolo Bientinesi,RWTH Aachen,https://www.aices.rwth-aachen.de/people/bientinesi,G6Z5vx8AAAAJ
+Paolo Bientinesi,RWTH Aachen University,https://www.aices.rwth-aachen.de/people/bientinesi,G6Z5vx8AAAAJ
```
Comment (emeryberger) [inline csrankings-p.csv]: Has a dual affiliation with Umea (https://hpac.cs.umu.se/~pauldj/) - please confirm that the RWTH appointment is >= 75% time.

File: csrankings-r.csv [hunk 3]
```diff
@@ -246,6 +246,7 @@ Ralf Möller 0002,Bielefeld University,http://ekvv.uni-bielefeld.de/pers_publ/pu
 Ralf Peeters,Maastricht University,https://www.maastrichtuniversity.nl/ralf.peeters,4gv6c1sAAAAJ
 Ralf Romeike,Freie Universitaet Berlin,https://www.mi.fu-berlin.de/en/inf/groups/ag-ddi/team/prof/index.html,NOSCHOLARPAGE
 Ralf Schenkel,University of Trier,https://www.uni-trier.de/index.php?id=17320,4gaNzroAAAAJ
+Ralf Schlüter, RWTH Aachen University,http://www-i6.informatik.rwth-aachen.de/~schluter/,JmuAC9oAAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv]: Spurious space after comma.

File: csrankings-t.csv [hunk 4]
```diff
@@ -448,7 +448,7 @@ Thomas R. Ioerger,Texas A&M University,http://faculty.cs.tamu.edu/ioerger,OfcXto
 Thomas Ranner,University of Leeds,https://eps.leeds.ac.uk/computing/staff/643/dr-thomas-ranner,dS8Hj_AAAAAJ
 Thomas Rauber,University of Bayreuth,http://www.ai2.uni-bayreuth.de/de/team/Rauber_Thomas,NOSCHOLARPAGE
 Thomas Ristenpart [Tech],Cornell University,http://tech.cornell.edu/people/thomas-ristenpart,MGVrVSIAAAAJ
-Thomas Rose,RWTH Aachen,http://dbis.rwth-aachen.de/cms/staff/rose,dR-qbA4AAAAJ
+Thomas Rose,RWTH Aachen University,http://dbis.rwth-aachen.de/cms/staff/rose,dR-qbA4AAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv]: Missing disambiguation suffix - https://dblp.org/pid/19/955-1.html.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Review by LKlinke — COMMENTED** [review]
finished change requests

**Comment by LKlinke** [comment]
I am unable to provide a successful commit validation without changing the institutes file (renaming RWTH Aachen to RWTH Aachen University)

**Comment by emeryberger** [comment]
I will push a change to the institutions list but there are still numerous issues with home pages returning 404s.

**Comment by emeryberger** [comment]
Look for the non institutions related issues:

```
Sanity checking the commit. Please check any issues raised here.
Processing Bastian Leibe,RWTH Aachen University,http://www.vision.rwth-aachen.de/persons,ZcULDB0AAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3ABastian%20Leibe:$%3A&format=json&c=10
  Checking homepage URL (http://www.vision.rwth-aachen.de/persons)
***Test failure for Bastian Leibe***.
Processing Bernhard Rumpe,RWTH Aachen University,http://www.se-rwth.de/~rumpe,nED1C[7](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:8)QAAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3ABernhard%20Rumpe:$%3A&format=json&c=10
  Checking homepage URL (http://www.se-rwth.de/~rumpe)
***Test failure for Bernhard Rumpe***.
Processing Christopher Morris 0001,RWTH Aachen University,https://chrsmrrs.github.io/,3f6_I[8](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:9)MAAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AChristopher%20Morris%200001:$%3A&format=json&c=10
  Checking homepage URL (https://chrsmrrs.github.io/)
***Test failure for Christopher Morris 0001***.
Processing Claudia Wagner 0001,RWTH Aachen University,http://claudiawagner.info/,NOSCHOLARPAGE
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AClaudia%20Wagner%200001:$%3A&format=json&c=10
  Checking homepage URL (http://claudiawagner.info/)
***Test failure for Claudia Wagner 0001***.
Processing Erich Grädel,RWTH Aachen University,https://logic.rwth-aachen.de/~graedel,4TbxnY0AAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AErich%20Gr%C3%A4del:$%3A&format=json&c=10
  Checking homepage URL (https://logic.rwth-aachen.de/~graedel)
  WARNING: Received error code 403.
  WARNING: Invalid homepage URL (https://logic.rwth-aachen.de/~graedel). Please provide a correct URL.
***Test failure for Erich Grädel***.
Processing Erika Ábrahám,RWTH Aachen University,http://www-i2.informatik.rwth-aachen.de/i2/eab,NOSCHOLARPAGE
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AErika%20%C3%81brah%C3%A1m:$%3A&format=json&c=10
  Checking homepage URL (http://www-i2.informatik.rwth-aachen.de/i2/eab)
***Test failure for Erika Ábrahám***.
Processing Erika Ábrahám-Mumm,RWTH Aachen University,http://www-i2.informatik.rwth-aachen.de/i2/eab,NOSCHOLARPAGE
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AErika%20%C3%81brah%C3%A1m%20Mumm:$%3A&format=json&c=10
  Checking homepage URL (http://www-i2.informatik.rwth-aachen.de/i2/eab)
***Test failure for Erika Ábrahám-Mumm***.
Processing Gerhard Lakemeyer,RWTH Aachen University,http://www-i5.informatik.rwth-aachen.de/gerhard,v0zpdqkAAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AGerhard%20Lakemeyer:$%3A&format=json&c=10
  Checking homepage URL (http://www-i5.informatik.rwth-aachen.de/gerhard)
***Test failure for Gerhard Lakemeyer***.
Processing Gerhard Woeginger,RWTH Aachen University,https://www.informatik.rwth-aachen.de/cms/informatik/Fachgruppe/Aktuell/Meldungen/~ukvwk/Trauer-um-Gerhard-Woeginger/,rtfJhO0AAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AUwe%20Naumann:$%3A&format=json&c=10
  Checking homepage URL (https://www.stce.rwth-aachen.de/people/uwe-naumann)
***Test failure for Uwe Naumann***.
Processing Walter Oberschelp,RWTH Aachen University,https://www.mathematik.rwth-aachen.de/go/id/bkbg/gguid/0x21CDE8D[9](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:10)CE85D51196710000F4B4937D/ikz/11/allou/1/lidx/1,NOSCHOLARPAGE
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AWalter%20Oberschelp:$%3A&format=json&c=[10](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:11)
  Checking homepage URL (https://www.mathematik.rwth-aachen.de/go/id/bkbg/gguid/0x21CDE8D9CE85D5[11](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:12)96710000F4B4937D/ikz/11/allou/1/lidx/1)
***Test failure for Walter Oberschelp***.
Processing Wil M. P. van der Aalst,RWTH Aachen University,http://www.vdaalst.com,aSZYyxIAAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AWil%20M%20P%20van%20der%20Aalst:$%3A&format=json&c=10
  Checking homepage URL (http://www.vdaalst.com)
***Test failure for Wil M. P. van der Aalst***.
Processing Wolfgang Prinz,RWTH Aachen University,http://dbis.rwth-aachen.de/cms/staff/prinz,QWWLOGQAAAAJ
  ERROR: This institution (RWTH Aachen University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AWolfgang%[20](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:21)Prinz:$%3A&format=json&c=10
  Checking homepage URL (http://dbis.rwth-aachen.de/cms/staff/prinz)
  WARNING: Received error code 404.
  WARNING: Invalid homepage URL (http://dbis.rwth-aachen.de/cms/staff/prinz). Please provide a correct URL.
***Test failure for Wolfgang Prinz***.
ERROR: This PR has too many diffs. Something probably went wrong.
Error: Process completed with exit code [25](https://github.com/emeryberger/CSrankings/actions/runs/6842949094/job/18604996713?pr=6432#step:6:26)5.
```

**Comment by LKlinke** [comment]
something is broken in the validation script. It said the URL for hector geffner is wrong, but try to access it and it will work properly.

**Comment by emeryberger** [comment]
Geffner works but these don't.

404: https://www.rwth-aachen.de/cms/root/Die-RWTH/Jobs-Ausbildung/Berufungen-an-der-RWTH-Aachen/Neuberufene/~kgfh/Univ-Prof-Torsten-Wolfgang-Kuhlen/lidx/1
404: https://dbis.rwth-aachen.de/dbis/staff/rose

File: csrankings-t.csv [hunk 5]
```diff
@@ -879,16 +879,14 @@ Torsten Höfler,ETH Zurich,https://htor.inf.ethz.ch,DdBvcBEAAAAJ
...
 Torsten Möller,University of Vienna,https://cs.univie.ac.at/torsten.moeller,3cGVmqsAAAAJ
 Torsten Muetze,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/torsten_mutze,9232pxwAAAAJ
 Torsten Mütze,University of Warwick,https://warwick.ac.uk/fac/sci/dcs/people/torsten_mutze,9232pxwAAAAJ
 Torsten Sattler,Czech Technical University,https://tsattler.github.io,jzx6_ZIAAAAJ
 Torsten Schaub,University of Potsdam,https://www.cs.uni-potsdam.de/wv/people/torsten,XknBv7MAAAAJ
 Torsten Söderström,Uppsala University,http://user.it.uu.se/~ts,NOSCHOLARPAGE
 Torsten Suel,New York University,http://engineering.nyu.edu/~suel,eQUn8ugAAAAJ
-Torsten W. Kuhlen,RWTH Aachen University,http://www.rwth-aachen.de/cms/root/Die-RWTH/Jobs-Ausbildung/Berufungen-an-der-RWTH-Aachen/Neuberufene/~kgfh/Univ-Prof-Torsten-Wolfgang-Kuhlen/lidx/1,O9ywg_UAAAAJ
-Torsten Wolfgang Kuhlen,RWTH Aachen University,http://www.rwth-aachen.de/cms/root/Die-RWTH/Jobs-Ausbildung/Berufungen-an-der-RWTH-Aachen/Neuberufene/~kgfh/Univ-Prof-Torsten-Wolfgang-Kuhlen/lidx/1,O9ywg_UAAAAJ
+Torsten W. Kuhlen,RWTH Aachen University,http://www.vr.rwth-aachen.de/person/1/,O9ywg_UAAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv:892]: Both of these must be listed:
* Torsten W. Kuhlen 
* Torsten Wolfgang Kuhlen

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-t.csv (hunk 5 above)
Comment (LKlinke) [reply csrankings-t.csv:892]: incorporated. Thanks for the pointer.

**Pull request merged**